File: attentive-android-sdk/src/main/java/com/attentive/androidsdk/Product.java
package com.attentive.androidsdk;

public class Product extends Event {
    private String productId;
    private String name;

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: attentive-android-sdk/src/main/java/com/attentive/androidsdk/internal/network/events/Product.kt
package com.attentive.androidsdk.internal.network.events

import kotlinx.serialization.Serializable

@Serializable
data class Product(
    val productId: String? = null,
    val variantId: String? = null,
    val name: String? = null,
    val variantName: String? = null,
    val imageUrl: String? = null,
    val categories: List<String>? = null,
    val price: String? = null,
    val quantity: Int? = null,
    val productUrl: String? = null,
)
